bin/reth/src/lib.rs

```diff
@@ -43,3 +44,38 @@ struct NetworkOpts {
     #[arg(long, value_delimiter = ',')]
     bootnodes: Option<Vec<NodeRecord>>,
 }
+
+/// Parameters for configuring the network more granularly via CLI
+#[derive(Debug, Args)]
+#[command(next_help_heading = "Rpc")]
+struct RpcServerOpts {
+    #[arg(long, default_value = "true")]
+    http: bool,
+
+    #[arg(long = "http.addr")]
+    http_addr: Option<String>,
+
+    #[arg(long = "http.port")]
+    http_port: Option<u128>,
+
+    #[arg(long = "http.api", value_delimiter = ',')]
+    http_api: Option<Vec<RethRpcModule>>,
```

Comment:
I'd like this to be `Option<RpcModuleConfig>` instead so we can do `http_api.unwrap_or_default()`

needs test for parsing, perhaps a `FromStr` impl for `RpcModuleConfig`